src/api/app/models/package.rb

```diff
@@ -1034,6 +1035,14 @@ def remove_linked_packages
     BackendPackage.where(links_to_id: self.id).delete_all
   end
 
+  def remove_devel_packages
+    Package.where(develpackage: self).each do |devel_package|
+      devel_package.develpackage = nil
```

Comment:
Broken Database? The links in the package that points to self should be broken and not silently discarded IMHO